skimage/transform/_geometric.py

```diff
@@ -571,23 +644,126 @@ def inverse(self, coords):
         return out
 
 
-class SimilarityTransform(ProjectiveTransform):
+class EuclideanTransform(ProjectiveTransform):
+    """2D Euclidean transformation of the form:
+
+    ..:math:
```

Comment:
@ahojnnes I think @sciunto is correct, in which case you should remove this tag...